server_model.js

```diff
@@ -134,6 +137,37 @@ Asana.ServerModel = {
         });
   },
 
+  /**
+   * Requests user type-ahead completions for a query.
+   */
+  userTypeAhead: function(workspace_id, query, callback, errback) {
```

Comment:
Total style nit: "typeahead" is one word, so we only capitalize the first letter when it must be capitalized.